cmake/patches/jconfig.h

```diff
@@ -0,0 +1,60 @@
+/* jconfig.h.  Generated from jconfig.cfg by configure.  */
```

Comment:
This file can be deleted. Nicolas branch mentions that it was for libjpeg and we do not use it anymore. His comment makes sense to me.

Reply:
Let's delete it if it's not used.